2022/Day7/Solve.py

```diff
@@ -0,0 +1,83 @@
+class Day_Six(object):
+  def day_six(self):
+    computer = self.parse("PuzzleInput.txt")
+    computer.calculate_size()
+    print("The answer to part one is", computer.sum_all_subdirectories_under_limit(100000))
+
+  def parse(self, puzzle_input):
+    current_directory = []
+    computer = Directory("dir /")
+    with open(puzzle_input, "r") as f:
+      for line in f:
+        if "cd /" in line:
+          continue
```

Comment:
shouldn't this reset current_directory?

Reply:
Oh! It should. It was clearly only called once, but yes, this should reset. I'll fix